### PR TITLE
OCPBUGS#29369: Update naming syntax for Operator groups

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.15

Issue: [OCPBUGS-29369](https://issues.redhat.com/browse/OCPBUGS-29369)

Link to docs preview: https://71496--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/understanding/olm/olm-understanding-operatorgroups#olm-operatorgroups-rbac_olm-understanding-operatorgroups

QE review:
- [x] QE has approved this change. (Acked in the JIRA ticket https://issues.redhat.com/browse/OCPBUGS-29369)

Additional information:

### CODE REVIEW

File: modules/olm-operatorgroups-rbac.adoc [hunk 1]
```diff
@@ -11,13 +11,13 @@ When an Operator group is created, three cluster roles are generated. Each conta
 |===
 |Cluster role |Label to match
 
-|`<operatorgroup_name>-admin`
+|`olm.og.<operatorgroup_name>-admin-<hash-value>`
```
Comment (stevsmit) [inline modules/olm-operatorgroups-rbac.adoc:14]: <hash_value> ? Is it a user-replaced value? https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

**Comment by stevsmit** [comment]
Just one question. LGTM otherwise.

**Comment by michaelryanpeter** [comment]
@bandrade PTAL.

**Pull request merged**